tests/drivers/adc/adc_api/boards/pico2_spe_rp2350a_m33.overlay

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2025 KWS Computersysteme Gmbh
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Pico2-SPE is compatible with the Pico-SPE , so reuse. */
+#include "rpi_pico.overlay"
```

Comment:
From the comment I assume that this was supposed to include `pico_spe.overlay`, though both the files have the same content anyway.